scripts/lib/install

```diff
@@ -555,6 +555,9 @@ cp -rT "$deploy_path"/prod-static/serve /home/zulip/prod-static
 chown -R zulip:zulip /home/zulip /var/log/zulip /etc/zulip/settings.py
 
 if ! [ -e /home/zulip/deployments/current/zulip-git-version ]; then
+    mkdir /srv/zulip.git
+    chown zulip:zulip /srv/zulip.git
+    su zulip -c 'git clone --bare --mirror /home/zulip/deployments/current /srv/zulip.git'
     su zulip -c 'cd /home/zulip/deployments/current && ./scripts/lib/update-git-upstream && ./tools/cache-zulip-git-version'
```

Comment:
For consistency, we should `git clone` after running `update-git-upstream`, right?

Reply:
We could -- that would get us the refs in `refs/remotes/upstream`, but not the `config` settings.  We could also run `update-git-upstream` in `/srv/zulip.git` after cloning, which would get us both of them -- but felt a little unnecessary since the first thing that `upgrade-zulip-from-git` does is run `update-git-upstream`.  And it seemed odd to fix up `upstream` but not `origin`, which felt like it became a bit of a rabbithold, with little actual value.